src/dmd/backend/evalu8.c

```diff
@@ -1599,63 +1599,55 @@ elem * evalu8(elem *e, goal_t goal)
 #if __DMC__
     case OPord:
         i++;
-    case OPunord:
         // BUG: complex numbers
         i ^= d1 !<>= d2;
         e->EV.Vint = i;
         break;
 
     case OPnlg:
         i++;
-    case OPlg:
         // BUG: complex numbers
         i ^= d1 <> d2;
```

Comment:
Infact this suggests that dmc supports nceg operators still, so definitely don't remove backend support.